tests/10-api.spec.js

```diff
@@ -1,18 +1,267 @@
 /*!
  * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
-import * as middleware from '..';
-import {authorizeZcapInvocation} from '..';
+import express from 'express';
+import {signCapabilityInvocation} from 'http-signature-zcap-invoke';
+import {Ed25519Signature2020} from '@digitalbazaar/ed25519-signature-2020';
+import {ZcapClient} from '@digitalbazaar/ezcap';
+import {httpClient, DEFAULT_HEADERS} from '@digitalbazaar/http-client';
+import {securityLoader} from '@digitalbazaar/security-document-loader';
+import zcapCtx from 'zcap-context';
+import {authorizeZcapInvocation} from '../lib';
+import {getInvocationSigner} from './helpers';
 
+const loader = securityLoader();
+loader.addStatic(zcapCtx.CONTEXT_URL, zcapCtx.CONTEXT);
+
+const documentLoader = loader.build();
+
+const TEST_SERVER_PORT = 5000;
+const BASE_URL = `http://localhost:${TEST_SERVER_PORT}`;
```

Comment:
Missed this -- still needs to be `https` if we can swing it.

Reply:
Updated cedbf5d